content/en/docs/project-user-guide/application-workloads/deployments.md

```diff
@@ -7,4 +7,291 @@ description: 'Kubernetes Deployments'
 weight: 2230
 ---
 
-TBD
+A Deployment controller provides declarative updates for Pods and ReplicaSets. You describe a desired state in a Deployment object, and the Deployment controller changes the actual state to the desired state at a controlled rate.
+
+## Prerequisites
+
+- You need to create a workspace, project and `project-regular` account. Please refer to the [Getting Started with Multi-tenant Management](../../../quick-start/create-workspace-and-project) if not yet.
+- You need to sign in with `project-admin` account and invite `project-regular` to enter the corresponding project if not yet. Please refer to [Invite Member](../../../quick-start/create-workspace-and-project#task-3-create-a-project).
+
+## Create a Deployment
+
+### Step 1: Click Create
+
+ Into **Workloads**, choose **DtatefulSets** and click **Create**
+
+![deployments](/images/docs/project-user-guide/workloads/deployments.png)
+
+### Step 2: Input Base Data
+
+![deployments](/images/docs/project-user-guide/workloads/deployments_form_1.png)
+
+### Step 3: Input Container Data
+
+![deployments](/images/docs/project-user-guide/workloads/deployments_form_2.png)
+
+#### Step 3.1: Container Setting
+
+Click the **Add Container Image** area.
+
+![deployments](/images/docs/project-user-guide/workloads/deployments_form_2_container_btn.png)
+
+You can input or select the image which in the public docker hub or private image repository you want to use.
+
+![deployments](/images/docs/project-user-guide/workloads/deployments_form_2_container_1.png)
+
+- **Image Name**
+
+  You can click the icon or input the image name to search it. kubesphere provide the Dockerhub image and your private image repository. If you want to use your private image repository, you should create a docker hub secret first in **Configurations** **Secrets**
```

Comment:
->
You can click the cube icon or input the image name to search it. KubeSphere provides Docker Hub images and your private image repository. If you want to use your private image repository, you should create a Docker Hub secret first in **Secrets** under **Configurations**.